sw/host/tests/manuf/cp_provision_init/BUILD

```diff
@@ -21,11 +26,16 @@ rust_binary(
     name = "cp_provision_init",
     srcs = [
         "src/main.rs",
+        "src/provisioning_command.rs",
         "src/provisioning_data.rs",
         ":lc_raw_unlock_token",
     ],
-    compile_data = [":provisioning_data"],
+    compile_data = [
+        ":provisioning_command",
+        ":provisioning_data",
+    ],
     rustc_env = {
+        "provisioning_command": "$(location :provisioning_command)",
         "provisioning_data": "$(location :provisioning_data)",
```

Comment:
Rather than the textual-include-via-env-var trick, rules_rust now supports generated sources.  You _might_ be able to list the `:provisioning_{command,data}` targets in `srcs`.  You'll still need `mod $name` statements for them in `main.rs`, and you'll need an empty module to satisfy `rustfmt`, but we might be able to do away with the textual include trick.

Its ok with me if you want to delay that experiment to a follow-up PR.